zmq.hpp

```diff
@@ -2224,6 +2224,14 @@ class monitor_t
         on_monitor_started();
     }
 
+    operator void *() ZMQ_NOTHROW { return handle(); }
+
+    operator void const *() const ZMQ_NOTHROW { return handle(); }
+
+    ZMQ_NODISCARD void *handle() ZMQ_NOTHROW { return _monitor_socket.handle(); }
+
+    ZMQ_NODISCARD const void *handle() const ZMQ_NOTHROW { return _monitor_socket.handle(); }
```

Comment:
Similar to the comment above, I'd use more specific names for these functions. We are in `monitor_t`, so `monitor` is probably redundant, but i'd use `socket_handle`.

Also, the `const` overload should be removed. There are no functions in the C zmq API that accept a `const void*` socket. I now they exist for other classes, but that's rather a legacy.

Reply:
Agreed.